src/test/java/Problem2Test.java

```diff
@@ -1,14 +1,39 @@
 import org.junit.Test;
 
-import static org.junit.Assert.*;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
 
 public class Problem2Test {
 
     // Example test
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+        //assertTrue(false);
+        //This is in ascending order
+        int input[][] = {
+                {},
+                {1, 4, 90, 3, 0, 13, 16},
+                {9, 8, 2, 1, 6, 0, 5, 3, 4, 7, 6},
+        };
+
+        int output[][] = {
+                {},
+                {0, 1, 3, 4, 13, 16, 90},
+                {0, 1, 2, 3, 4, 5, 6, 6, 7, 8, 9}
+        };
+
+        assertEquals(input.length, output.length);
+        assertEquals(input.length, output.length);
+
+
+        for (int i = 0; i < input.length; i++) {
+            Problem2.bubbleSort(input[i]);
+            //System.out.println(i);//print
+            System.out.println("Sorted Array: " + Arrays.toString(input[i]));
```

Comment:
-5: missing checks for result

Reply:
There are 3 parts for any test:

1. setup
2. calling the function being tested
3. verify

this is missing the 3rd step because there's no assert to check. printing out the result doesn't mean checking. here it's the automation that's checking the result. this will always show as PASS. compare this with the problem1Test and see how it's done there.

Reply:
in future homework, no checks will cause you to loose most points for a homework. make sure you fix this. feel free to ping me on discord if you have questions.